src/main/scala/is/hail/rvd/RVD.scala

```diff
@@ -143,14 +143,14 @@ trait RVD {
       it.map { rv => f(c, rv) }
     })
 
-  def map[T](f: (RegionValue) => T)(implicit tct: ClassTag[T]): RDD[T] = rdd.map(f)
+  def map[T](f: (RegionValue) => T)(implicit tct: ClassTag[T]): RDD[T] = crdd.map(f).run
```

Comment:
Hmm.  How's this used?  I'm slightly happier if this return a ContextRDD[C, T].  Same with the map functions below.

Reply:
These functions upset me as well. This change fixes a bug I anticipated wherein the `f` may be used to "stabilize" or "bring on heap" the region values. This ain't kosher because in `rdd.map(f)` the stabilization happens after the lifetime is over.

I started addressing your comment but it grew quickly into quite a few changes. I've removed the change from this PR and will create a follow up PR to modify these methods to return `ContextRDD[RVDContext, T]`